src/arrayMethodLastIndexOf.js

```diff
@@ -5,8 +5,23 @@
  *
  */
 function applyCustomLastIndexOf() {
-  [].__proto__.lastIndexOf2 = function() {
-    // write code here
+  [].__proto__.lastIndexOf2 = function(item, position) {
+    let pos;
```

Comment:
pos is not very clear. I'd recommend renaming to 'adjustedPosition' or smth like that. It's not a lot better but it indicated we are calculating it somehow.